src/vm/hints/hint_utils.rs

```diff
@@ -223,6 +292,19 @@ pub fn get_integer_from_relocatable_plus_offset<'a>(
     )))
 }
 
+pub fn insert_integer_at_relocatable_plus_offset(
+    int: BigInt,
+    relocatable: &Relocatable,
+    field_offset: usize,
+    vm: &mut VirtualMachine,
+) -> Result<(), VirtualMachineError> {
+    let new_pos =
+        MaybeRelocatable::from((relocatable.segment_index, relocatable.offset + field_offset));
```

Comment:
Let's just implement the trait `Add` for `Relocatable` and `usize`.

Reply:
resolved: https://github.com/lambdaclass/cleopatra_cairo/pull/319/commits/dc04f99079a2c3fcea0e1bab5848a107484e5bdf